README.md

```diff
@@ -135,6 +135,18 @@ flux bootstrap github \
 Another option is to copy the images from the ControlPlane registry to your organization's registry
 and use the `--registry` flag to point to your registry.
 
+Copying an image from the ControlPlane registry to your organization's registry can be done with the following commands:
+
+```shell
+crane copy ghcr.io/controlplaneio-fluxcd/distroless/helm-controller:v0.37.4 <your-registry>/controlplaneio-fluxcd/helm-controller:v0.37.4
+```
```

Comment:
Crane can sync the whole repo with `--all-tags`, no need to specify the version. I don't think we should be recommending oras as it has no such capabity.

Reply:
Also please write a a for loop here like this:

```
FLUX_CONTROLLERS=(
"source-controller"
"kustomize-controller"
"helm-controller"
"notification-controller"
"image-reflector-controller"
"image-automation-controller"
)

for i in ${!FLUX_CONTROLLERS[@]}; do
  crane copy ghcr.io/controlplaneio-fluxcd/distroless/${FLUX_CONTROLLERS[$i]}  <your-registry>/${FLUX_CONTROLLERS[$i]}
done
```